src/app/services/RBA/rba.service.ts

```diff
@@ -59,6 +60,7 @@ export class RBAService {
       this.permissions.createNewLabel= true;
       this.permissions.editLabel= true;
       this.permissions.deleteLabel= true;
+      this.permissions.addContributors= true;
```

Comment:
This enables Admins and managers from all the organizations' to add the contributors, 
We need to show add Contributors only to the admins of Worktez.